pyas2/models.py

```diff
@@ -376,7 +429,7 @@ def send_message(self, header, payload):
         # Send the message to the partner
         try:
             response = requests.post(
-                self.partner.target_url, auth=auth, headers=header, data=payload)
+                self.partner.target_url, auth=auth, headers=header, data=payload, verify=self.partner.https_verify_ssl)
```

Comment:
what line lengths are you using, max will be 100

Reply:
120 was mine, changed to 100 and have this in next commit.